src/components/HeroImage/styles.scss

```
@@ -0,0 +1,71 @@
+// Mobile portrait
+.hero {
+	position: fixed;
+	width: 100%;
+	height: 100%;
+	top: 0;
+	right: 0;
+	z-index: 0;
+}
+
+	.glow {
+		position: absolute;
+		width: 1em;
+		height: 1em;
+		top: 0;
+		left: 50%;
+		font-size: 100vw;
+
+	}
+		.glow::before {
```

```suggestion
		&::before {
```
and move it into the `.glow` definition above